src/components/calcite-combobox/calcite-combobox.tsx

```diff
@@ -106,16 +147,20 @@ export class CalciteCombobox {
 
   @Event() calciteComboboxChipDismiss: EventEmitter;
 
-  @Listen("calciteComboboxItemChange") calciteComboboxItemChangeHandler(
-    event: CustomEvent<HTMLCalciteComboboxItemElement>
-  ): void {
+  @Listen("click", { target: "document" })
+  documentClickHandler(event: Event): void {
+    const target = event.target as HTMLElement;
+    this.setInactiveIfNotContained(target);
+  }
+
+  @Listen("calciteComboboxItemChange")
+  calciteComboboxItemChangeHandler(event: CustomEvent<HTMLCalciteComboboxItemElement>): void {
     this.toggleSelection(event.detail);
   }
 
-  @Listen("calciteChipDismiss") calciteChipDismissHandler(
-    event: CustomEvent<HTMLCalciteChipElement>
-  ): void {
-    this.textInput.focus();
+  @Listen("calciteChipDismiss")
+  calciteChipDismissHandler(event: CustomEvent<HTMLCalciteChipElement>): void {
```

Comment:
Nitpick: Same as above `calcite` is implied.

Reply:
I'll leave as is since it wasn't modified.